coral/src/app/features/topics/details/messages/TopicMessages.tsx

```diff
@@ -61,7 +61,7 @@ function TopicMessages() {
       getTopicMessages({
         topicName,
         consumerGroupId: "notdefined",
```

Comment:
Here's the suggestion for the update :D

Since the string "notdefined" represents a default consumer group id being configured in cluster api, I would create a variable for that. 

This variable should go into the file:
`coral/src/domain/topic/topic-types.ts` 


It could be:

`const TOPIC_MESSAGE_DEFAULT_USER_GROUP_ID = "notdefined"`

(note: the ALL_CAPS is a pattern that came from the olden days of JS, wenn there was no `const` :D and is still used in a lot of code bases to signal that a value is constant and immutable on the first glance)

and needs to be exported with 

`export { TOPIC_MESSAGE_DEFAULT_USER_GROUP_ID };` at the end (all other exports are types, so they can't go in there)

Then, in `coral/src/domain/topic/index.ts` you import this const and export it again (outside of the `domain` directory, we only ever import from those index files).

Then you can import it and use it here. 

This way, we don't have a magical random string as a value, but it's a readable variable related to the domain of topic-messages.